webui/react/src/constants/states.ts

```diff
@@ -158,6 +158,7 @@ export const jobStateToLabel: { [key in JobState]: string } = {
   [JobState.SCHEDULED]: 'Scheduled',
   [JobState.SCHEDULEDBACKFILLED]: 'ScheduledBackfilled',
   [JobState.QUEUED]: 'Queued',
+  [JobState.UNSPECIFIED]: 'Queued',
```

Comment:
I haven't looked up where we use this but maybe a label that communicates we don't know might be less confusing? we don't expect to get the unspecified state IIRC

Reply:
I agree that we shouldn't end up with this state, but I think that all of the Enums are supposed to include Unspecified as the null/default state

Reply:
> I think that all of the Enums are supposed to include Unspecified as the null/default state

👍🏼 . Do you think it'd make sense to give this a more explicit "we don't know what's up" value